src/models/dataElementsSet.ts

```diff
@@ -382,7 +386,48 @@ export default class DataElementsSet {
             .value();
         const dataElementsUpdated = this.updateSelected(newSelected);
 
-        return { selectionInfo, dataElements: dataElementsUpdated };
+        const selectionInfoInOtherSectors = deInOtherSectors.map(r => r.selectionMessage);
+
+        return {
+            selectionInfo: {
+                messages: selectionInfo.messages?.concat(selectionInfoInOtherSectors),
+            },
+            dataElements: dataElementsUpdated,
+        };
+    }
+
+    private getDataElementsInOtherSectors(selectedIds: string[], sectorId: string) {
+        return selectedIds.flatMap(dataElementId => {
+            const sectorsIds = Object.keys(this.data.selected);
+            const deInOtherSectors = _(sectorsIds)
+                .map(currentSectorId => {
+                    if (currentSectorId === sectorId) return undefined;
+                    const deIds = this.data.selected[currentSectorId];
+                    const isInOtherSector = deIds.includes(dataElementId);
+                    if (!isInOtherSector) return undefined;
+                    const dataElement = this.data.dataElementsBySector[currentSectorId].find(
+                        de => de.id === dataElementId
+                    );
+                    if (!dataElement) return undefined;
+                    const pairedDe = _(dataElement.pairedDataElements).first();
+                    if (!pairedDe) return undefined;
+
+                    return {
```

Comment:
The example in the ticket indeed shows "indicators A and B cannot be selected", but I guess that a DE selection should also not be allowed when there is are no paired DEs. Confirm with @adrianq

Reply:
@eperedo @tokland That is correct. We don't care if the DE is paired or not. We just don't want to have the data element in two different sectors